### PR TITLE
Add Vet Clinic Database Schema, Data, and Queries

### PR DESCRIPTION
**Pull request by MussieTeka**
This pull request adds the Vet Clinic database schema, sample data, and relevant SQL queries.

#### Changes Made:
- Created the "vet_clinic" database and defined the "animals" table with the required columns.
- Inserted sample data for four animals into the "animals" table.
- Implemented SQL queries to address various tasks, including finding animals by name, birth year, neutering status, escape attempts, and weight range.

Additional Notes:
- The SQL scripts for creating the schema, inserting data, and the queries have been included in the repository.
- Screenshots of the query results have been attached for reference below.

Thank you for reviewing my work.

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @MussieTeka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] All data insertion done in the `data.sql` file :1st_place_medal: 
- [ ] All queries are supported with various screenshots :rocket: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -286,6 +165,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
[the remark this answers is not in the document]
Comment (MussieTeka) [reply README.md:166]: Hi @addod19,
Thank you for your feedback. The MIT license link is not broken, and I have already created and included the license file in the commit history. It is one of the changes I am merging into the "Dev" branch. Please review the commit history for confirmation. Let me know if you have any further questions or need clarification.

[the remark this answers is not in the document]
Comment (MussieTeka) [reply README.md:166]: Hi @addod19,
Thank you for your feedback. The MIT license link is not broken, and I have already created and included the license file in the commit history. It is one of the changes I am merging into the "Dev" branch. Please review the commit history for confirmation.

**Review by alexander16108 — APPROVED** [review]
Hi @MussieTeka  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 Correct Git-flow followed 💯
 All data insertion done in the data.sql file 🥇
 All queries are supported with various screenshots 🚀

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**